src/features/landing/components/Header/NavLink/NavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { Link, ListItem, Stack, List } from '@mui/material';
+import React from 'react';
+
+function NavLink({ links }: { links: { id: number; value: string }[] }) {
+  return (
+    <Stack component="nav">
+      <List sx={{ display: 'flex' }}>
+        {links.map(link => (
+          <ListItem key={link.id}>
+            <Link href="#" sx={{ color: '#57595B' }} underline="none">
```

Comment:
need to use Link component from next.js, create custom Link component and use it across all application